discord/ext/commands/cooldowns.py

```diff
@@ -111,6 +112,8 @@ def _bucket_key(self, msg):
             return (msg.guild or msg.author).id
         elif bucket_type is BucketType.channel:
             return msg.channel.id
+        elif bucket_type is BucketType.member:
+            return ((msg.guild or msg.author).id, msg.author.id)
```

Comment:
I would rather it be `((msg.guild and msg.guild.id), msg.author.id)` i.e. the first key is `None` if there's no guild.